signing_service/src/constants.py

```diff
@@ -0,0 +1 @@
+SIGNING_SERVICE_DEFAULT_PORT = 8888
```

Comment:
Looks like I have left my comment in the wrong pull request. See https://github.com/golemfactory/concent/pull/634/files#r202398516

> Middleman uses 8888 for accepting connections from Concent (the internal port). You should use the external port there.

Reply:
But is is not defined yet neither in any pull request and it doesn't have default in https://github.com/golemfactory/concent/issues/616. Which one should I use then ?

Reply:
Use 9055 (see https://github.com/golemfactory/concent/pull/619/files#r202727409).

Reply:
:+1: done